test/integration/aws/default/build/cloudtrail.tf

```diff
@@ -186,6 +186,9 @@ resource "aws_cloudtrail" "trail_1" {
   cloud_watch_logs_group_arn = "${aws_cloudwatch_log_group.trail_1_log_group.arn}"
   cloud_watch_logs_role_arn  = "${aws_iam_role.cloud_watch_logs_role.arn}"
   kms_key_id                 = "${aws_kms_key.trail_1_key.arn}"
+  provisioner "local-exec" {
```

Comment:
This looks like an unrelated change - could you remove it?

Reply:
Sorry about that! That is actually a fix I had to do in order to get the aws setup to work on my system.  It looks like it just needed to wait a little for other resources to finish before it builds that resource.